### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by YonathanMeguira**
truncating values emitted back to form if step < 1

### CODE REVIEW

File: lib/src/slider/slider.component.ts [hunk 1]
```diff
@@ -49,9 +49,12 @@ export class DatoSliderComponent extends BaseCustomControl implements AfterViewI
   @Input()
   set options(data: SliderOptions) {
     /*
-         * set default values
-          */
+             * set default values
```
Comment (NetanelBasal) [inline lib/src/slider/slider.component.ts]: add test

**Comment by NetanelBasal** [comment]
@YonathanMeguira Add the missing test and pay attention that the core project replaced with new one. First, pull from the master.

**Comment by NetanelBasal** [comment]
I merged it into the new project. Don't do anything, please.

**Comment by YonathanMeguira** [comment]
ok

**Pull request closed without merging**